frontend/src/components/Review/ReviewImageUploader/ReviewImageUploader.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Heading, Spacing, Text } from '@fun-eat/design-system';
+import React, { useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+const ReviewImageUploader = () => {
+  const [reviewImage, setReviewImage] = useState('');
+  const theme = useTheme();
+
+  const uploadReviewImage = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const uploadReviewImage: ChangeEventHandler<HTMLInputElement> = (event) => {
```

핸들러 함수의 타입을 가져오는건 어떤가요?

Reply:
그럼 이렇게 작성하는걸로 컨벤션 업데이트 할게요~